README.md

```diff
@@ -1,76 +1,37 @@
-<p>
-  <a href="https://www.tinybird.co/join-our-slack-community"><img alt="Slack Status" src="https://img.shields.io/badge/slack-chat-1FCC83?style=flat&logo=slack"></a>
-</p>
+# CI Flows for Tinybird Data Projects
 
-# CI Flows for Tinybird Projects
+[![Slack][slack_img]][slack_community_url]
+[![License][repo_license_img]][repo_license_url]
 
-Collection of configuration files that enable CI flows for Tinybird projects. 
+A collection of configuration files that enable CI flows for Tinybird data projects.
 
-## Features
+## ⚡️ Quick start
 
-- Templates for popular CI tools like GitHub Actions and GitLab CI
+These configuration files have been created iterating you Tinybird data projects using CI & CD jobs.
```

Comment:
I'm not 100% what are you referring here. Maybe you want to say: ".. have been created to iterate your Tinybird..."

Reply:
Correct!